storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -42,10 +42,10 @@ include::modules/persistent-storage-byok.adoc[leveloffset=+1]
 If there is no encrypted key defined in the storage class, only set `encrypted: "true"` in the storage class. The AWS EBS CSI driver uses the AWS managed alias/aws/ebs, which is created by Amazon EBS automatically in each region by default to encrypt provisioned storage volumes. In addition, the managed storage classes all have the `encrypted: "true"` setting.
```

Comment:
@smunje1 does this need to say "The Amazon EBS CSI driver uses..."

Reply:
I agree that this text should be updated to "Amazon EBS".

Reply:
Same as the [comment](https://github.com/openshift/openshift-docs/pull/59650#discussion_r1274403918)